components/WalletContainer.tsx

```diff
@@ -14,6 +13,7 @@ import {
 
 const config = createConfig({
   chains: [mainnet],
+  ssr: true,
```

Comment:
Will this only work on the backend side? or both?
How does the NodeJS renderer know that the UI is connected to a wallet?

Reply:
From the Wagmi [docs](https://wagmi.sh/react/guides/ssr):

> Turning on the ssr property means that content from the external stores will be hydrated on the client after the initial mount.
> 
> As a result of turning on the ssr property, external persistent stores like localStorage will be hydrated on the client after the initial mount.
> 
> This means that you will still see a flash of "empty" data on the client (e.g. a "disconnected" account instead of a "reconnecting" account, or an empty address instead of the last connected address) until after the first mount, when the store hydrates.

It seems that `useSkipFirstRender` does something similar but I think it's better to use the solution provided by Wagmi since it works out-of-box and we can stop worrying about hydration.

A possible definitive solution is the use of cookies to let the server know the client is already connected:

> In order to persist data between the server and the client, you can use cookies.

But I don't think it's a priority for now.

Reply:
I'd personally see a blank screen followed by a loading spinner, rather than an incorrect "disconnected" state. Even if wagmi ships it, I still prefer handling things explicitly on our components rather than needing to interpret that "disconnected = disconnected or rehydrating" 

My 5c

Reply:
For me is working just fine 😯 Maybe is a cache from the non-ssr version or some old dependencies.

The problem I see is that we have a lot of components using `skipFirstRender` and in most cases we are avoiding to render the whole component. That means the server is sending mostly empty pages to the clients.
On the other hand, managing a temporal "disconnected state" give us more room to decide how to render it.

I think we can use the Wagmi solution for now (if we confirm is working 😅) and in a following PR implement cookies so we can be more "smart" about what the server sends back.